Documentation: Update prometheus getting started doc (#60803)
grafana/grafana: docs/sources/getting-started/get-started-grafana-prometheus.md

{{< docs/shared "getting-started/first-step.md" >}}

#### 1. Download Prometheus and node_exporter

Download the following components:

- [Prometheus](https://prometheus.io/download/#prometheus)
- [node_exporter](https://prometheus.io/download/#node_exporter)

> Prometheus, like Grafana, can be installed on many different operating systems. Refer to the [Prometheus download page](https://prometheus.io/download/), which lists all stable versions of Prometheus components.

Reviewer: ```suggestion
Like Grafana, you can install Prometheus on many different operating systems. Refer to the [Prometheus download page](https://prometheus.io/download/) to see a list of stable versions of Prometheus components.
```

#### 2. Install Prometheus node_exporter

> Install node_exporter on all hosts you want to monitor. In this guide we will assume you are installing it locally. Prometheus node_exporter is a widely used tool that exposes system metrics. For instructions on how to install node_exporter, refer to the [Installing and running the node_exporter](https://prometheus.io/docs/guides/node-exporter/#installing-and-running-the-node-exporter) section in Prometheus documentation.

Reviewer: ```suggestion
Install node_exporter on all hosts you want to monitor. This guide shows you how to install it locally. 

Prometheus node_exporter is a widely used tool that exposes system metrics. For instructions on installing node_exporter, refer to the [Installing and running the node_exporter](https://prometheus.io/docs/guides/node-exporter/#installing-and-running-the-node-exporter) section in the Prometheus documentation.
```

> When running node_exporter locally, you can check that it is exporting metrics by navigating to `http://localhost:9100/metrics`.

Reviewer: ```suggestion
When you run node_exporter locally, navigate to `http://localhost:9100/metrics` to check that it is exporting metrics.
```

> **Note**: The instructions in the referenced topic are intended for Linux users. You may have to alter the instructions slightly depending on your operating system. For example, if you are on Windows, use the [windows_exporter](https://github.com/prometheus-community/windows_exporter) instead.

#### 3. Install and configure Prometheus

> 1. After downloading Prometheus [here](https://prometheus.io/download/#prometheus), extract it and navigate to the directory.

Reviewer: ```suggestion
1. After [downloading Prometheus](https://prometheus.io/download/#prometheus), extract it and navigate to the directory.
```

   ```
   tar xvfz prometheus-*.tar.gz
   cd prometheus-*
   ```

1. Locate the `prometheus.yml` file in the directory.

> 1. Configure Prometheus to monitor the hosts where you installed node_exporter. In order to do this, modify Prometheus's configuration file. By default, Prometheus looks for the file `prometheus.yml` in the current working directory. This behavior can be changed via the `--config.file` command line flag. For example, some Prometheus installers use it to set the configuration file to `/etc/prometheus/prometheus.yml`. Here is an example of the code you will need to add. Notice that static configs targets are set to `['localhost:9100']` to target node-explorer when running it locally.

Reviewer: ```suggestion
1. Modify Prometheus's configuration file to monitor the hosts where you installed node_exporter. 

By default, Prometheus looks for the file `prometheus.yml` in the current working directory. This behavior can be changed via the `--config.file` command line flag. For example, some Prometheus installers use it to set the configuration file to `/etc/prometheus/prometheus.yml`. 

The following example shows you the code you should add. Notice that static configs targets are set to `['localhost:9100']` to target node-explorer when running it locally.
```

   ```
   tar xvfz prometheus-*.tar.gz
   cd prometheus-*
   ```

1. Locate the `prometheus.yml` file in the directory.

1. Modify Prometheus's configuration file to monitor the hosts where you installed node_exporter.

By default, Prometheus looks for the file `prometheus.yml` in the current working directory. This behavior can be changed via the `--config.file` command line flag. For example, some Prometheus installers use it to set the configuration file to `/etc/prometheus/prometheus.yml`.

The following example shows you the code you should add. Notice that static configs targets are set to `['localhost:9100']` to target node-explorer when running it locally.

```
 # A scrape configuration containing exactly one endpoint to scrape from node_exporter running on a host:
 scrape_configs:
     # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
     - job_name: 'node'

     # metrics_path defaults to '/metrics'
     # scheme defaults to 'http'.

       static_configs:
       - targets: ['localhost:9100']
```

1. Start the Prometheus service:

   ```
    ./prometheus --config.file=./prometheus.yml
   ```

> 1. Confirm that Prometheus is running by navigating to `http://localhost:9090`.

Reviewer 2: Maybe we want to call out that 9090 is the default port, but could change depending on their prometheus installation? Either that or add the prometheus config above that includes the 9090 target, so it's obvious where that's coming from (not everyone will know that 9090 is the default port for prom like we do)

        # scheme defaults to 'http'.

          static_configs:
          - targets: ['localhost:9100']
   ```

1. Start the Prometheus service:

   ```
    ./prometheus --config.file=./prometheus.yml
   ```

> 1. Confirm that Prometheus is running by navigating to `http://localhost:9090`. You can see that the node_exporter metrics have been delivered to Prometheus. Next we will send them to Grafana.

Reviewer: ```suggestion
1. Confirm that Prometheus is running by navigating to `http://localhost:9090`. 

You can see that the node_exporter metrics have been delivered to Prometheus. Next, the metrics will be sent to Grafana.
```

#### 4. Configure Prometheus for Grafana

> When running Prometheus locally, there are two ways to configure Prometheus for Grafana. You may use a hosted Grafana instance at [Grafana Cloud](https://grafana.com/) or run Grafana locally. This guide will describe configuring Prometheus in a hosted Grafana instance on Grafana Cloud.

Reviewer: ```suggestion
When running Prometheus locally, there are two ways to configure Prometheus for Grafana. You can use a hosted Grafana instance at [Grafana Cloud](https://grafana.com/) or run Grafana locally. 

This guide describes configuring Prometheus in a hosted Grafana instance on Grafana Cloud.
```

1. Sign up for [https://grafana.com/](https://grafana.com/auth/sign-up/create-user). Grafana gives you a Prometheus instance out of the box.

> SHOW PICTURE HERE

Reviewer: @bohandley , images are stored in the website repo and then you can add a reference to that image here. For image guidelines, take a look at https://grafana.com/docs/writers-toolkit/writing-guide/image-guidelines/.
